src/Render/Program.cs

```diff
@@ -89,13 +89,48 @@ internal static class Program {
 		}
 
 		private static string GetTitle( MarkdownDocument doc ) {
-			var inline = doc.Descendants<HeadingBlock>().Single(
-				h => h.Level == 1
-			).Inline;
-			var titleLiteral = inline.Descendants<LiteralInline>().Single();
+
+			HeadingBlock headingBlock = GetSingleTitleOrThrow( doc );
+
+			LiteralInline titleLiteral = GetUnformattedContentOrThrow( headingBlock );
+
 			return titleLiteral.Content.ToString();
 		}
 
+		private static LiteralInline GetUnformattedContentOrThrow(HeadingBlock headingBlock) {
+			var literalInlines =
+				headingBlock
+					.Inline
+					.Descendants<LiteralInline>()
+					.ToArray();
+
+			if( literalInlines.Length > 1 ) {

```

Comment:
I moved away from `SingleOrDefault` because it throws if there are multiple entries.

Reply:
Yeah, `SingleOrDefault` has very unintuitive behaviour without reading the docs (`default` if 0, the value if 1, throw if >1).